src/hw-2/index.ts

```diff
@@ -0,0 +1,75 @@
+import {getAutoSuggestUsers, getUserIndex} from './utils';
+import {User} from './types';
+import {validation} from './validation';
+import * as core from 'express-serve-static-core';
+
+const express = require('express');
+const app = express();
+const router: core.Router = express.Router();
+
+const PORT = 3000;
+const {v1: uuidv4} = require('uuid');
+const userStore: User[] = [];
+
+app.listen(PORT, () => {
```

Comment:
Are you running the server before you've set up its routes? That's strange.

Reply:
Fixed. Thanks.